From: Boyuan Yang <byang@debian.org>
Date: Sat, 1 Feb 2020 00:53:56 -0500
Subject: QNodeEditor.pro: Specify version

---
 QNodeEditor.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/QNodeEditor.pro b/QNodeEditor.pro
index 66ca5b0..0ee4e82 100644
--- a/QNodeEditor.pro
+++ b/QNodeEditor.pro
@@ -1,4 +1,4 @@
 TARGET =        QNodeEditor
 TEMPLATE =      lib
-VERSION =       1.0.0
+VERSION =       2.1.5
 include(QNodeEditor.pri)
